### DISCUSSION TITLE
[Question][Hw7] Help me understand the type problems in example1.c

### DISCUSSION THREAD

**Discussion by sjoon2455** [prosyslab-classroom/cs524-program-analysis#391] [Q&A]
Hi,

Could someone help me understand what type problems are in the given test/example1.c ?
Document indicates something could be potentially wrong with types of variable ```w``` and ```v```, which I couldn't really get.
Why is each variable's type a problem here?
Is it because w is not explicitly casted to float, and v is not explicitly casted in line 7?

![image](https://user-images.githubusercontent.com/38874408/144987978-96617596-c20e-4732-af97-10bfe7d88210.png)

Thanks in advance (_ _)

**Comment by ShinWonho** [comment]
example1.ll is slightly different from example1.c

![image](https://user-images.githubusercontent.com/50018375/144989683-bf188da9-a319-46ad-97f4-1ab06c593ed1.png)

Here, rather than storing 12 at %v, %div is stored at %v, which causes the type error.

**Comment by sjoon2455** [comment]
@ShinWonho Oh!!! Thank you!
